answers/Constants/316e9667-5913-442b-a51a-de213ae066fc/AddConstantDefinitionInClassTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+class Cylinder
+{
+    const PI = 3.1416;
+    private $radius;
+    private $height;
+
+    public function __construct($radius, $height)
+    {
+        $this->radius = $radius;
+        $this->height = $height;
+    }
+
+    public function getRadius()
+    {
+        return $this->radius;
+    } 
+
+    public function getHeight()
+    {
+        return $this->height;
+    } 
+
+    public function area()
+    {
+        return 2 * self::PI * $this->getRadius() * ($this->getRadius() + $this->getHeight());
+    }
+
+    public function volume()
+    {
+        return self::PI * $this->getRadius() * $this->getRadius() * $this->getHeight();
+    }
+
+    public function display()
+    {
+        echo "Area: " . $this->area();
+        echo "\nVolume: " . $this->volume();
```

Comment:
@cerilacanda Can we move the `\n` to the end of the first `echo`? It's most common to append the new line at the end of the line. Not at the beginning of the line.

Reply:
@TenshiLyle I think this is possible my purpose why I add `\n` at the beginning of the second echo is to avoid another concatenation in the first echo. Please let me check and do some changes. Thank you.